### ISSUE TITLE
Camera Undo

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#2645]
I hate to loose hours of work. And now i did once more. We should investigate if there is a way to add the camera object to the undo system.

**Comment by iyadahmed** [comment]
hmm can this issue be explained more

**Comment by Draise14** [comment]
You can undo a camera if you select it and undo it with CTRL+Z.. But it's finicky. 

Basically if you move the view, you probably can press CTRL+Y and it will only undo the camera movement that just happened. Nothing else undos. 

This is an old feature from other software.

**Comment by ReinerBforartists** [comment]
Sheesh, i never knew that this is possible. I was always under the impression that there is no undo for the camera at all. Thanks Draise :D

Okay, then we need to rethink this task. I still think it should be possible to press ctrl z without to extra select the camera object.

**Comment by Draise14** [comment]
Yeah, but it only works for camera... doesn't work with the view outside the camera.

**Comment by ReinerBforartists** [comment]
Indeed. There is the stored views addon. But you first have to make a save ...

**Comment by ReinerBforartists** [comment]
Thought about this a bit more. The initial task was about the camera object. But this is already part of undo. I just didn't know it. And i don't see a way to improve this undo any further. We could write a bug report why the camera movement is not undone immediately when you perform undo. But i don't have high hopes for a fix or an improvement. The workaround is too easy: select the cam, then try again.

And for the view camera i don't see a way to add it to the undo system at all without to add lots of roadblocks to the workflow. Default undo steps is 25. Now do a few dances around the object in the viewport ...

There is already the stored views addon with which you can work around in case you really need and have to. 

Long story short, here is nothing left to do. Closing. Thanks again Draise for telling me how the undo for a cam works :)
